app/lib/search/backend.dart

```diff
@@ -187,14 +188,12 @@ class PopularityStorage {
 
   void _updateLatest(Map raw) {
     final Map<String, int> rawTotals = {};
+    final popularity = new PackagePopularity.fromJson(raw);
     final List items = raw['items'];
```

Comment:
Remove this `items`, since we already have `popularity.items` now (use it below in the logging message)?

Reply:
ack